docs/paywall3.md

```diff
@@ -750,6 +751,17 @@ Here are listed and defined the Paywall 3.0 standalone functions:
 
 Table 4
 
+### `donations`
+
+This HTML class can be added to any HTML element. The `asset-id` represents the ID of a specific InPlayer asset. Usage example:
```

Comment:
This is a great and elaborate description, we should however provide it under the table in the _**Standalone Functionalities**_ section. 

Let's move this section there